SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Security Policy
+
+Full details of the Automattic Security Policy can be found on [automattic.com](https://automattic.com/security/).
+
+## Supported Versions
+
+### Web
+
+Only the latest version of Calypso is supported.
+
+### WordPress Desktop
+
+[ADD WP DESKTOP STUFF HERE]
```

Comment:
I'm assuming this will replicate what we've currently written above for Web, but I'll defer to @nsakaimbo to confirm.